### DISCUSSION TITLE
Adopt Bun? 😆

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#433] [Ideas]
Bun was just released and it's amazing 🤩

Both `import { createElement } from 'react'` and `const = require('lodash')` on the same file? Yes please!

https://www.youtube.com/watch?v=BsnCpESUEqM

**Comment by kentcdodds** [comment]
I've been keeping a sharp eye on bun. I'd welcome [an example](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md). The biggest issue is going to be Node.js compatibility and windows support.

I definitely plan on switching to bun in the future if they stay on track with where they're at right now.

**Reply by andrecasal** [reply]
I've seen you ask for examples with very little results (if I'm not mistaken). I believe this is because the process is somewhat cumbersome. Would it be possible to just take the current Epic Stack, create a PR and convert **that** into an example?

**Reply by kentcdodds** [reply]
I think I don't get results because it takes effort and people are busy.

I don't know how creating an example is any more work than making a PR.

The reason I ask for an example instead is because if I decide to not include it in the template then the PR is just closed. But if you make an example at least people benefit from having the example.

**Reply by andrecasal** [reply]
I didn't mean have the PR be the example. I meant create a fork from the PR.

**Reply by kentcdodds** [reply]
Sorry, I don't know what you mean 😅 maybe you could show me? 😁

**Comment by excessivecoding** [comment]
I got a try at it but it looks like Bun doesn't support Remix completely yet as indicated in the [roadmap](https://github.com/oven-sh/bun/issues/159). at least it seems to be on the radar for the near future 🤞
